client/src/routes/wind/WindChart.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable react/jsx-handler-names */
```

Comment:
please remove all eslint disable directives, all these rules were added by intention, and correct your code accordingly